app/workers/commit_monitor_handlers/commit_range/rubocop_checker.rb

```diff
@@ -22,29 +22,123 @@ def perform(branch_id, _new_commits)
   private
 
   def process_branch
-    @results = merged_linter_results
-    unless @results["files"].blank?
-      diff_details = diff_details_for_merge
-      @results     = RubocopResultsFilter.new(results, diff_details).filtered
-    end
-
-    replace_rubocop_comments
+    run
   rescue GitService::UnmergeableError
     nil # Avoid working on unmergeable PRs
   end
 
-  def rubocop_comments
-    MessageBuilder.new(results, branch).comments
+  def run
+    proton_messages = pronto_result
+    comment_body = pronto_format(proton_messages)
+    comments = create_comment(comment_body)
+    replace_pronto_comments(comments)
+  end
+
+  def create_comment(body)
+    header = ""
+
+    header << pronto_tag
+    header << "#{"Commit".pluralize(commits.length)} #{commit_range_text} checked with ruby #{RUBY_VERSION} and:\n\n"
+    header << versions_table
+    header << "\n\n"
+
+    [header + body]
+  end
+
+  def replace_pronto_comments(pronto_comments)
+    pr_url = "https://github.com/#{fq_repo_name}/pull/#{pr_number}"
+    logger.info("Updating #{pr_url} with Pronto comment.")
+
+    GithubService.replace_comments(fq_repo_name, pr_number, pronto_comments) do |old_comment|
+      pronto_comment?(old_comment)
+    end
+  end
+
+  def pronto_tag
+    "<pronto />"
+  end
+
+  def pronto_comment?(comment)
+    comment.body.start_with?(pronto_tag)
+  end
+
+  def versions_table
+    str = "Pronto Runners | Version | Linters | Version\n--- | --- | --- | ---\n"
+
+    versions.each do |ver|
+      v = ver.to_a
+      str << v[0].first + " | " + v[0].last + " | " + v[1].first + " | " + v[1].last + "\n"
+    end
+
+    str.strip
```

Comment:
I think this can be done more simply (acccurately?) with something like the CSV class...

```ruby
CSV.generate(:col_sep => " | ") do |csv|
  csv << %w["Pronto\ Runners Version Linters Version]
  csv << %w[--- --- --- ---]
  versions.each { |ver| csv << ver.to_a.flatten }
end
```